python/lesson3/tutorial.md

```diff
@@ -56,42 +56,40 @@ Now that we have defined a parameter, the function will raise an error if no par
 We can enhance the function to run with a default name if one is not provided. Let's do that by specifying a default value.
 
     >>> def print_name(name='Bart'):
-    ...     print "My name is " + name
+    ...     print("My name is " + name)
     ...
     >>> print_name("Lisa")
     My name is Lisa
     >>> print_name()
     My name is Bart
 
-
 ## Different kinds of functions
 
-A function can be put into one of two categories. Fruitful functions, and void functions. So far we have only created void functions, these are functions that perform tasks but don't return a value. Fruitful functions return a value, a familiar example is the range function we used earlier that returned a list. Let's change our function from void to fruitful.
+A function can be put into one of two categories. "Fruitful" functions, and "void" functions. So far we have only created void functions, these are functions that perform tasks but don't return a value. Fruitful functions return a value, a familiar example is the `len()` function we used earlier that returned an integer. Let's change our function from void to fruitful.
 
-    >>> def print_name(name='Bart'):
+    >>> def get_name_statement(name='Bart'):
     ...    return "My name is " + name
     
 Now the function returns a value that we can work with. We could store this in a value to be used later. Let's try storing it as a value, and using that value as a parameter in another `print_name()` function call.
 
-    >>> printed_name = print_name("Lisa")
-    >>> print print_name(printed_name)
+    >>> name_statement = get_name_statement("Lisa")
+    >>> print print_name(name_statement)
```

Comment:
have you double checked that the `print_name` function still works at this point?